elixir/apps/domain/lib/domain/config.ex

```diff
@@ -136,6 +136,10 @@ defmodule Domain.Config do
     feature_enabled?(:traffic_filters)
   end
 
+  def relay_admin_enabled? do
```

Comment:
What do you think about slightly different naming here?
```suggestion
  def self_hosted_relays_enabled? do
```

Reply:
Sounds good 👍